### PR TITLE
MXNet pre-quantized BERT

### PR DESCRIPTION
**Pull request by anijain2305**
MXnet pre-quantized BERT model - https://gluon-nlp.mxnet.io/examples/sentence_embedding/bert.html#Quantize-the-model

Features added in this PR

* Support for Tensor quantization for MXNet Dense operator
* Support for Channel quantization for MXNet Dense operator
* Adding channel wise support for dequantize op
* Support softmax use_length for axis=-1

### CODE REVIEW

**Comment by anijain2305** [comment]
@icemelon9 @eric-haibin-lin @shoubhik 

Please review

File: python/tvm/relay/frontend/nnvm_common.py [hunk 1]
```diff
@@ -57,9 +60,53 @@ def _impl(inputs, attrs):
...
+
+                # Input data is now 2D, we can set the axis = 1
+                axis = 1
+            elif data_ndims > 2:
+                raise error.OpNotImplemented(\
+                        "Operator softmax with use_length=True is supported only for axis -1")
+
+            res = _op.sequence_mask(data=data,
+                                    valid_length=length,
+                                    mask_value=float(min_value("float").value),
```
Comment (icemelon9) [inline python/tvm/relay/frontend/nnvm_common.py:100]: It'll be better to use the dtype of data instead of "float".

Comment (anijain2305) [reply python/tvm/relay/frontend/nnvm_common.py]: Done

**Comment by anijain2305** [comment]
@icemelon9 Can you please manage this PR?

**Comment by icemelon9** [comment]
Thanks @anijain2305

**Pull request merged**